package.json

```diff
@@ -13,5 +13,8 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11"
+  },
```

Comment:
We already have this in the server folder and it doesn't look like you've added any scripts. Is this necessary?